server/models/Contest.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose')
+
+const ContestSchema = new mongoose.Schema({
+    title: {
+        type: String
+    },
+
+    description: {
+        type: String,
+    },
+
+    prizeAmount: {
```

Comment:
Fields which are necessary for a contest can be marked as required while defining the schema.